query/src/main/scala/filodb/query/exec/aggregator/AbsentRowAggregator.scala

```diff
@@ -0,0 +1,34 @@
+package filodb.query.exec.aggregator
+
+import filodb.core.query._
+import filodb.memory.format.RowReader
+
+/**
+  * Map: Every sample is mapped to itself
+  * ReduceMappedRow: Same as ReduceAggregate since every row is mapped into an aggregate
+  * ReduceAggregate: Accumulator maintains the min. Reduction happens by choosing one of currentMin, or the value.
+  * Present: The min is directly presented
+  */
+object AbsentRowAggregator extends RowAggregator {
```

Comment:
It's surprising this is required as part of this PR. Could you leave a quick note here about why this wasn't needed until now?

Reply:
Also, is this already covered by existing tests?

Reply:
File deleted.